home/tests/import-export-data/content2.csv

```diff
@@ -24,12 +24,12 @@ Reply with a number to take part in a *free* self-help program created by WHO.
 
 1. Quit tobacco 🚭
 _Stop smoking with the help of a guided, 42-day program._
-2. Manage your stress 🧘🏽‍♀️
+2. Manage your stress
```

Comment:
I think `content2.csv` is used in several places, so let's rather add a modified copy (to use for whichever test this is for) instead of modifying `content2.csv` itself.

Reply:
So what we're doing here is just removing the non-printable characters from this CSV, because otherwise the import-export-compare tests fail because those characters have been removed. I've said that it's okay to modify this CSV in this way, since we have tests specific to the removal of chars, so we don't need those chars also in this CSV.